M2/libraries/flint/Makefile.in

```diff
@@ -15,12 +15,9 @@ CONFIGOPTIONS += --enable-assert
 CFLAGS += -O0 -fno-unroll-loops 
 endif
 CFLAGS += -std=c90 -pedantic-errors
-# the flint configure script does not accept CPPFLAGS
-# CONFIGURECMD =  LIB_DIRS=$(LIBRARIESDIR)/lib ./configure  --with-gc --with-blas --disable-tls
-CONFIGURECMD =  ./bootstrap.sh && \
-	LIB_DIRS=$(LIBRARIESDIR)/lib ./configure --without-blas  \
-			--prefix='$(PREFIX)' --disable-shared CC='$(CC)' \
-			CFLAGS='$(CFLAGS) $(CPPFLAGS)'
+
+PRECONFIGURE = ./bootstrap.sh
+CONFIGOPTIONS += --without-blas --disable-shared
```

Comment:
Is `--without-blas` necessary here?

Reply:
Not sure -- it's currently just a carryover from before. I'll play around with it at some point